docs/source/user_guide/model_understanding.ipynb

```diff
@@ -565,6 +565,84 @@
     "                                        include_shap_values=True, output_format=\"dataframe\")\n",
     "report"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Force Plots\n",
+    "Force plots can be generated to predict single or multiple rows for binary, multiclass and regression problem types.  Here's an example of predicting a single row on a binary classification dataset.  The force plots show the predictive power of each of the features in making the negative (\"Class: 0\") prediction and the positive (\"Class: 1\") prediction. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import shap\n",
+    "\n",
+    "from evalml.demos import load_breast_cancer\n",
+    "from evalml.model_understanding.force_plots import graph_force_plot\n",
+    "from evalml.pipelines import BinaryClassificationPipeline\n",
+    "\n",
+    "X, y = load_breast_cancer()\n",
+    "\n",
+    "\n",
+    "pipeline = BinaryClassificationPipeline(component_graph = ['Simple Imputer', 'Random Forest Classifier'])\n",
+    "pipeline.fit(X, y)\n",
+    "\n",
+    "shap.initjs() # Allows plots to be visualized in Jupyter notebook\n",
+    "rows_to_explain = [0] # Should be a list of integer indices of the rows to explain.\n",
+    "\n",
+    "results = graph_force_plot(pipeline, rows_to_explain=rows_to_explain, training_data=X.df, y=y)\n",
+    "\n",
+    "for result in results:\n",
+    "    for cls in result:\n",
+    "        print(\"Class:\", cls)\n",
+    "        display(result[cls][\"plot\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here's an example of a force plot explaining multiple predictions on a multiclass problem.  These plots show the force plots for each row arranged as consecutive columns that can be ordered by the dropdown above.  Clicking the column indicates which row explanation is underneath."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evalml.demos import load_wine\n",
+    "from evalml.model_understanding.force_plots import graph_force_plot\n",
+    "from evalml.pipelines import MulticlassClassificationPipeline\n",
+    "\n",
+    "X, y = load_wine()\n",
+    "\n",
+    "pipeline = MulticlassClassificationPipeline(component_graph = ['Simple Imputer', 'Random Forest Classifier'])\n",
+    "pipeline.fit(X, y)\n",
```

Comment:
Style nit-pick: if its important to show a multiclass example for force plots, I suggest you:
* Rename the variable name of the binary classification pipeline which all the rest of the model understanding tools use to `pipeline_binary`
* At the top of the notebook where we define the binary pipeline, move this code up there so we also define `pipeline_multiclass`
* Then below you can use `pipeline_multiclass`

The reason I like this is that it keeps the notebook's narrative super simple. First train models, then visualize.

Reply:
gucci

Reply:
I made the changes to the pipeline names, but some of the visualization, particularly with the binary pipelines, is spread out all over the notebook.  I renamed them, but kept them in the same spot for now.